docs/templates/reachview/base-mode.mdx

```diff
@@ -72,31 +78,46 @@ In the Data rate row you can find an estimation of bytes/sec for 1 satellite whe
 |1107|SBAS MSM|37.5|
 |1117|QZSS MSM|42|
 |1127|BeiDou MSM|17.78|
-
+{%else%}
+|RTCM3 messages|Message type|Data rate, bytes/sec (1 satellite, 1Hz)|
+|:---:|:---:|:---:|
+||**Minimal required messages**||
+|1006|ARP station coordinate   | 27|
+|1074|GPS MSM4  | 19.38 |
+||**Optional messages for other GNSS **||
+|1084|GLONASS MSM4| 19.38 |
+|1094|Galileo MSM4| 9.58 |
+|1124|BeiDou MSM4| 18.9 |
+{%endif%}
 
 Here is some information about each message from RTCM STANDARD 10403.3<sup>[1](#myfootnote1)</sup>:
-
+{%if model == 'Module' or model == 'M+' or model == 'RS/RS+' %}
 - **Message Type 1002 and 1010** contain L1 RTK GPS and GLONASS raw satellite data respectively. GPS 1002 message is mandatory to use due to RTKLIB code. You could turn on GLONASS 1010 message if you want to use this GNSS.
 
 - **Message Type 1006** provides the earth-centered, earth-fixed (ECEF) coordinates of the antenna reference point (ARP) for a stationary reference station and the height of the ARP above a survey monument. It is the second mandatory message to turn on after GPS 1002.
 
 - **Messages 1097 (GALILEO), 1107 (SBAS), 1117 (QZSS), 1127 (BeiDou)** are MSM7 (Multiple Signal Messages). MSM7 are high precision messages which contains a complete set of RINEX observations with extended resolution. That means that you should turn on only one message of choosen system to get all data about it.
+{%else%}
+- **Message Type 1006** provides the earth-centered, earth-fixed (ECEF) coordinates of the antenna reference point (ARP) for a stationary reference station and the height of the ARP above a survey monument. It is the mandatory message to turn on.
```

Comment:
Why does this message appear here twice? You can just move it outside the if-case